flagging_site/templates/api/index.html

```diff
@@ -26,4 +27,52 @@ <h2>Available Endpoints</h2>
         <li><a href="{{ url_for('api.reachesapi') }}" target="_blank">Flagging Model Outputs</a></li>
         <li><a href="{{ url_for('api.boathousesapi') }}" target="_blank">Boathouses</a></li>
     </ul>
+    <h2>Sample code to call API</h2>
+    <ul>
+        <li>Calling Flagging Model Outputs</li>
+        <ul>
+            <li>
+                using Python
+                <pre class="prettyprint">
+                    import pandas as pd   
+                    url = "http://127.0.0.1:5000/api/v1/model"
+                    model_output = pd.read_json(url)
+                    print(model_output.head())
+                </pre>
+            </li>
+            <li>
+                using R
+                <pre class="prettyprint">
+                    library(jsonlite)
+                    url = "http://127.0.0.1:5000/api/v1/model"
+                    model_output <- fromJSON(url)
+                    print( head(model_output) )
+                </pre>
+            </li>
+        </ul>
+        <li>Calling Boathouse Metadata</li>
+        <ul>
+            <li>
+                using Python<br/>
+                <pre class="prettyprint">
+                    import pandas as pd
+                    url = "{{ url_for('api.index', _external=True) }}v1/boathouses"
+                    data = pd.read_json(url)
+                    print(data.head())
```

Comment:
I tested this and I'm glad to have learned about `read_json()` working with URLs, never knew that. Unfortunately it looks like it only works for super simple schemas, and the data doesn't load as intended. There are two options:

(1) In the API, remove the key `"boathouses": [{ ... }, {...}, ... ]` and simply return `[{ ... }, {...}, ... ]`. I'm personally not a fan of that because I think it's a good idea to label what your values represent in APIs, and also the more verbose schema is more flexible (e.g. you can add additional keys for metadata if you want, not that we will but it's nice to keep the option available as a rule of thumb).

(2) show slightly different example code that works with the existing API. I prefer that personally. Here's what I came up with:

```python
import pandas as pd
import requests
url = "{{ url_for('api.index', _external=True) }}v1/boathouses"
records = requests.get(url).json()["boathouses"]
data = pd.DataFrame(records)
print(data.head())
```

And here's the output:

<img width="561" alt="image" src="https://user-images.githubusercontent.com/31971762/95001695-686cdf00-059a-11eb-9da1-b7b8c08b3ac8.png">